data/routes/en.yml

```diff
@@ -1,61 +1,15 @@
 placeholders:
   jss-main:
-    - componentName: ContentBlock
+    - componentName: Topnavigation
       fields:
-        heading: Welcome to Sitecore JSS
-        # to author content in YAML, use _multi-line values_ (prefixed with | + endline)
-        # as long as the same indent is preserved at the beginning of each line, no escaping
-        # is required at all in the value, making it easy to read
-        content: |
-          <p>Thanks for using JSS. Here are some resources to get you started:</p>
-
-          <h3><a href="https://jss.sitecore.com" rel="noopener noreferrer">Documentation</a></h3>
-          <p>The official JSS documentation can help you with any JSS task from getting started to advanced techniques.</p>
-
-          <h3><a href="/styleguide">Styleguide</a></h3>
-          <p>The JSS styleguide is a living example of how to use JSS, hosted right in this app.
-          It demonstrates most of the common patterns that JSS implementations may need to use,
-          as well as useful architectural patterns.</p>
-
-          <h3><a href="/graphql">GraphQL</a></h3>
-          <p>JSS features integration with the Sitecore GraphQL API to enable fetching non-route data from Sitecore - or from other internal backends as an API aggregator or proxy.
-          This route is a living example of how to use an integrate with GraphQL data in a JSS app.</p>
-
-          <div class="alert alert-dark">
-            <h4>This app is a boilerplate</h4>
-            <p>The JSS samples are a boilerplate, not a library. That means that any code in this app is meant for you to own and customize to your own requirements.</p>
-            <p>Want to get change the lint settings? Do it. Want to read manifest data from a MongoDB database? Go for it. This app is yours.</p>
-          </div>
-
-          <div class="alert alert-dark">
-            <h4>How to start with an empty app</h4>
-            <p>To remove all of the default sample content (the Styleguide and GraphQL routes) and start out with an empty JSS app:</p>
-            <ol>
-              <li>Delete <code>/src/app/components/styleguide-*</code> and <code>/src/app/components/graph-ql-*</code></li>
-              <li>Delete <code>/sitecore/definitions/components/styleguide-*</code>, <code>/sitecore/definitions/templates/styleguide-*</code>, and <code>/sitecore/definitions/components/graph-ql-*</code></li>
-              <li>Delete <code>/data/component-content/Styleguide</code></li>
-              <li>Delete <code>/data/content/Styleguide</code></li>
-              <li>Delete <code>/data/routes/styleguide</code> and <code>/data/routes/graphql</code></li>
-              <li>Delete <code>/data/dictionary/*.yml</code></li>
-            </ol>
-          </div>
+        heading: Bookstore
+        shopping_cart:
+          src: assets/svg/shopping-cart.png
+          alt: shopping cart
+        hamburger: 
```

Comment:
would suggest naming it hamburger-button, or else seems there would be hamburger image :D

Reply:
Changed